assignment/scripts/function-practice.js

```diff
@@ -11,34 +11,39 @@ function hello() {
 // Call the function to test
 console.log('Test - should say "Hello World!"', hello());
 
+console.log(hello())
 
 // 2. Function to return an personalized hello, using the `name` argument.
 //    for example 'Hello, Jo!', or 'Hello, Stacy!'
 function helloName( name ) {
-  return;
+  return `Hello, ${name}!`;
 }
 // Remember to call the function to test
 
+console.log(helloName('Josh')) 
 
 // 3. Function to add two numbers together & return the result
-function addNumbers( firstNumber ) {
+function addNumbers( firstNumber, secondNumber ) {
   // return firstNumber + secondNumber;
+  return firstNumber+secondNumber; 
```

Comment:
Would love to see spaces around the operators.

```suggestion
  return firstNumber + secondNumber; 
```